tests/mainnet.rs

```diff
@@ -43,7 +43,7 @@ fn mainnet_blocks_from_rpc() {
             .unwrap();
         let rpc_storage = RpcStorage::new(provider, BlockId::number(block_number - 1));
         let db = CacheDB::new(&rpc_storage);
-        common::test_execute_alloy(db.clone(), block.clone(), None, true);
+        common::test_execute_alloy(db.clone(), block.clone(), None, true, true);
```

Comment:
After removing `must_succeed` we can keep only one `true` here.

Reply:
related to https://github.com/risechain/pevm/pull/134#discussion_r1627930717